test/test_helper.exs

```diff
@@ -1,4 +1,4 @@
-ExUnit.start()
+ExUnit.start(exclude: [:slow])
```

Comment:
There should be a separate CI task that runs these with `--only slow` I think

Reply:
Btw shouldn't we be running tests with exla?